Libraries/LibGC/Ptr.h

```diff
@@ -183,6 +183,14 @@ class Ptr {
 
     operator T*() const { return m_ptr; }
 
+    Ref<T> release_nonnull()
```

Comment:
I don't know that a "release" API really makes sense here. It's used in `OwnPtr` and `RefPtr` due to ownership mechanics, but that doesn't apply here - `GC::Ptr` / `GC::Ref` aren't owning objects.

So maybe just:
```c++
Ref<T> as_nonnull()
{
    ASSERT(m_ptr);
    return *m_ptr;
}
```